lib/livebook_web/live/settings_live.ex

```diff
@@ -139,6 +141,22 @@ defmodule LivebookWeb.SettingsLive do
               add_env_var_path={Routes.settings_path(@socket, :add_env_var)}
             />
           </div>
+          <!-- Protocol  -->
+          <div class="flex flex-col space-y-4">
+            <h2 class="text-xl text-gray-800 font-medium pb-2 border-b border-gray-200">
+              Custom Protocol
+            </h2>
+            <p class="mt-4 text-gray-700">
+              Add custom protocol to this Livebook instance, which are available inside your notebooks.
+              Enter the protocols below and separate with a comma.
+            </p>
```

Comment:
```suggestion
            <h2 class="text-xl text-gray-800 font-medium pb-2 border-b border-gray-200">
              Hyperlink protocols
            </h2>
            <p class="mt-4 text-gray-700">
              Livebook sanitizes links in Markdown, allowing only a few standard protocols
              by default (such as http and https). You can configure additional protocols that
              should be supported. Type a comma-separated list of protocols below.
            </p>
```

Reply:
Let's adjust the module/variable to "hyperlink protocols", so it is clearly linked to Markdown content. I was thinking "allowed/extra hyperlink protocols", but sounds like too much :)